### ISSUE TITLE
Remove requirement to use "pi" user.

### ISSUE THREAD

**Issue by brian32768** [f5vmr/svxlinkbuilder#10]
Remove all hardcoded references to /home/pi

**Comment by f5vmr** [comment]
In the initial compilation I strongly recommended that the user be "pi" to standardise a build, in order that I can help with debugging later. That said, for individual preference later I have pointed out on the facebook page svxlink amateur radio users that 'sudo nmtui' can be deployed to change user if required. Although this is not foolproof and requires the /etc/hosts to be edited. Alternatively sudo raspi-config can be used more effectively. In the svxlinkbuilder script, I found that I had to keep track of where the shell was going in and out of the functions, so adding /home/pi was as way to reset (in my head) the pointer. I had removed some of them. There are other ways to run the script as another user. But in this case uniformity was key.
